feature/policy_forwarding/decapsulation/otg_tests/ip_guev1_static_decap_subnet_range/ip_guev1_static_decap_subnet_range_test.go

```diff
@@ -0,0 +1,642 @@
+	ate.OTG().StopTraffic(t)
+	otgutils.LogFlowMetrics(t, ate.OTG(), config)
+}
+
+func protocolStart(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, config gosnappi.Config) {
+	ate.OTG().StartProtocols(t)
+	otgutils.WaitForARP(t, ate.OTG(), config, "IPv4")
+	otgutils.WaitForARP(t, ate.OTG(), config, "IPv6")
+	verifyBGPTelemetry(t, dut)
+}
+
+func verifyTrafficFlow(t *testing.T, ate *ondatra.ATEDevice, flowName string, trafficShouldPass bool) {
+	recvMetricV4 := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(flowName).State())
+
+	framesTxV4 := recvMetricV4.GetCounters().GetOutPkts()
+	framesRxV4 := recvMetricV4.GetCounters().GetInPkts()
+
+	if trafficShouldPass {
+		t.Logf("traffic validation for flow %s. Expecting Traffic TX = RX.", flowName)
+		if framesTxV4 == 0 {
+			t.Error("No traffic was generated and frames transmitted were 0")
+		} else if framesRxV4 == framesTxV4 {
+			t.Logf("Traffic validation successful for [%s] FramesTx: %d FramesRx: %d", flowName, framesTxV4, framesRxV4)
+		} else {
+			t.Errorf("Traffic validation failed for [%s] FramesTx: %d FramesRx: %d", flowName, framesTxV4, framesRxV4)
+		}
+	} else {
+		t.Logf("traffic validation for flow %s. Expecting Traffic Loss", flowName)
+		if framesTxV4 == 0 {
+			t.Error("No traffic was generated and frames transmitted were 0")
+		} else if framesRxV4 == 0 {
+			t.Logf("PASS: Traffic Validation is successful as no packets received at the destination as Expected")
+		} else {
+			t.Error("FAIL: Traffic Validation is failed as no packets expected at the destination ")
+		}
+	}
+}
+
+// startCapture starts the capture on the otg ports
+func startCapture(t *testing.T, ate *ondatra.ATEDevice) {
+	otg := ate.OTG()
+	cs := gosnappi.NewControlState()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.START)
+	otg.SetControlState(t, cs)
+}
+
+// stopCapture starts the capture on the otg ports
+func stopCapture(t *testing.T, ate *ondatra.ATEDevice) {
+	otg := ate.OTG()
+	cs := gosnappi.NewControlState()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.STOP)
+	otg.SetControlState(t, cs)
+}
+
+func enableCapture(t *testing.T, config gosnappi.Config, port string) {
+	config.Captures().Clear()
+	t.Log("Enabling capture on ", port)
+	config.Captures().Add().SetName(port).SetPortNames([]string{port}).SetFormat(gosnappi.CaptureFormat.PCAP)
+}
+
+func processCapture(t *testing.T, ate *ondatra.ATEDevice, port string) string {
+	otg := ate.OTG()
+	bytes := otg.GetCapture(t, gosnappi.NewCaptureRequest().SetPortName(port))
+	time.Sleep(captureWait * time.Second)
+	pcapFile, err := os.CreateTemp("", "pcap")
+	if err != nil {
+		t.Errorf("ERROR: Could not create temporary pcap file: %v\n", err)
+	}
+	if _, err := pcapFile.Write(bytes); err != nil {
+		t.Errorf("ERROR: Could not write bytes to pcap file: %v\n", err)
+	}
+	defer pcapFile.Close()
+	return pcapFile.Name()
+}
+
+func verfiy_policer_matched_packets(t *testing.T, dut *ondatra.DUTDevice) uint64 {
```

Comment:
```suggestion
func verify_policer_matched_packets(t *testing.T, dut *ondatra.DUTDevice) uint64 {
```

Reply:
Suggested change and the existing line looks similar. Could you please let me what change has to be done here.